lib/Applify.pm

```diff
@@ -46,9 +48,18 @@ sub app {
     $self->_exit('version');
   }
 
-  $application_class = $self->{application_class} ||= $self->_generate_application_class($code);
-  $app = $application_class->new(
-    {map { my $k = $self->_option_to_attr($_); $k => $self->_upgrade($k, $options{$_}) } keys %options});
+  $self->{application_class} ||= $self->_generate_application_class($code);
+
+  eval {
+    $app
```

Comment:
Sorry, but I don't get it. Can you tell me exactly what to change?

Reply:
localise the `$Type::Tiny::DD` to a code ref that translates `$_[0]` which is the value (can be a reference) within the `eval {}` and `Type::Tiny` will use that rather than it's default. Which, while informative, isn't the best for happy users.

See https://metacpan.org/pod/Type::Tiny#Package-Variables